drivers/serial/uart_sam0.c

```diff
@@ -377,6 +377,93 @@ static void uart_sam0_rx_timeout(struct k_work *work)
 
 #endif
 
+static int uart_sam0_configure(struct device *dev,
+					const struct uart_config *new_cfg)
```

Comment:
Nit: Indentation

To be consistent with the rest of the file, we should have:
```
static int uart_sam0_configure(struct device *dev,
			       const struct uart_config *new_cfg)
```

Note that the tab width is 8.